js/plugins/dotprompt/src/index.ts

```diff
@@ -14,14 +14,53 @@
  * limitations under the License.
  */
 
+import { genkitPlugin, PluginProvider } from '@genkit-ai/core';
+
 import { readFileSync } from 'fs';
 import { basename } from 'path';
-
 import { defineDotprompt, Dotprompt } from './prompt.js';
-import { lookupPrompt } from './registry.js';
+import { loadPromptFolder, lookupPrompt } from './registry.js';
 
 export { defineDotprompt, Dotprompt };
 
+export interface DotpromptPluginOptions {
+  promptDir: string;
+}
+
+let pluginOptions: DotpromptPluginOptions;
```

Comment:
This is weird global state stuff. Will this go away once you fix async init?

Reply:
Yes, we don't need this any more!

Reply:
Can you remove it then?

Reply:
Yes, it's removed. I just forgot to push it.